packages/mediacenter/kodi/scripts/kodi-config

```diff
@@ -66,6 +66,11 @@ else #arm
   echo "MALLOC_MMAP_THRESHOLD_=8192" >> /run/libreelec/kodi.conf
 fi
 
+# required for inputstreamhelper to detect TLS 64-bytes support
+if [ -n "$(uname -m | grep arm)" ] || [ "$(uname -m)" == "aarch64" ]; then
```

Comment:
`LIBC_WIDEVINE_PATCHLEVEL` should also only apply to project Amlogic. This can be done like with:
```suggestion
if [ "$(uname -r)" = "3.14.29" ]; then
```

Reply:
I thought all ARMs were affected, including ng-builds. Why should it be limited to the amlogic-3.14 kernel only?

Reply:
```
...Amlogic-ng as latest can be upgraded to CE-19 or CE-20...
```
...which include all needed fixes for already 3 month now.

Reply:
Okay, but it still affects all ARMs, including 64-bit kernels (arm64 *, aarch64).

This will not make it mandatory to build official images for the Amlogic-ng devices, but anyone who wants to do this for themselves should have their source patched in the same way. 
For example: LE didn't build their 9.2.8 images other than PRi2. It is the same situation.